Crude oil PR

mkhraisha opened the pull request:
This PR adds Crude Oil:

Some issues with testing similar to #89 looking for a cleaner solution for it.

Will be issuing a second PR for Gas/ QP-INBOND soon.

mkhraisha commented:
Theres new conflicts due to merge of cmtr, will be updating soon

mprorock reviewed (approved):
will need to pull latest changes and merge with your branch before we can merge the pr

OR13 commented:
^ all these files:

```
docs/contexts/traceability-v1.jsonld
docs/index.html
docs/intermediate.json
```

should be git ignored and built on PR merges... and we would have way less conflicts.

mkhraisha commented:
will update gitignore in my PR then

mprorock commented:
@mkhraisha while you are in there, do you mind running npm lint and cleaning up anything (line lengths, etc) that may be breaking ci?

mkhraisha commented:
There appears to be a number of errors and there are changes  to the generate-credential-examples folder, so i'll be updating the oil samples to reflect them.

mkhraisha commented:
Fixed lint issues and added the files to `gitignore` however they were still tracked so i removed them from cache and recommited. This should remove the files from your local repos on next pull.

OR13 commented on .gitignore:
```diff
@@ -1,4 +1,7 @@
\ No newline at end of file
+.idea
+docs/contexts/traceability-v1.jsonld
+docs/index.html
```
```suggestion
```

OR13 replied:
we need to keep this until we add a github action.

OR13 commented:
Actually I am going to take this PR and do some much needed cleanup
